src/Catalog/Helpers/XsltHelper.cs

```diff
@@ -40,22 +42,17 @@ public string LowerCase(string original)
 
         public string NormalizeVersion(string original)
```

Comment:
I chose to wrap thinly here because `XsltHelper` needs to be called via an instance method. `NuGetVersionUtility` is static so it's easily consumable by different jobs.